packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -77,8 +81,12 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
 
   public componentDidUpdate(prevProps: ICommandBarProps, prevStates: ICommandBarState) {
     if (!this._commandItemWidths) {
-      this._updateItemMeasurements();
-      this._updateRenderedItems();
+      // Asynchronously update command bar layout to eliminate forced synchronous reflow
+      var timer = setTimeout(() => {
```

Comment:
Would it be better to request animation frame here, rather than using setTimeout?